lms/djangoapps/courseware/tests/test_discussion_module.py

```diff
@@ -1,132 +0,0 @@
-# -*- coding: utf-8 -*-
-"""Test for Discussion Xmodule functional logic."""
-import ddt
-from django.core.urlresolvers import reverse
-from mock import Mock
-from . import BaseTestXmodule
-from course_api.blocks.tests.helpers import deserialize_usage_key
-from course_blocks.tests.helpers import EnableTransformerRegistryMixin
-from courseware.module_render import get_module_for_descriptor_internal
-from student.tests.factories import UserFactory, CourseEnrollmentFactory
-from xmodule.discussion_module import DiscussionModule
-from xmodule.modulestore import ModuleStoreEnum
-from xmodule.modulestore.django import modulestore
-from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
-from xmodule.modulestore.tests.factories import ToyCourseFactory, ItemFactory
-
-
-@ddt.ddt
-class DiscussionModuleTest(BaseTestXmodule, EnableTransformerRegistryMixin, SharedModuleStoreTestCase):
-    """Logic tests for Discussion Xmodule."""
-    CATEGORY = "discussion"
-
-    def test_html_with_user(self):
-        discussion = get_module_for_descriptor_internal(
-            user=self.users[0],
-            descriptor=self.item_descriptor,
-            student_data=Mock(name='student_data'),
-            course_id=self.course.id,
-            track_function=Mock(name='track_function'),
-            xqueue_callback_url_prefix=Mock(name='xqueue_callback_url_prefix'),
-            request_token='request_token',
-        )
-
-        fragment = discussion.render('student_view')
-        html = fragment.content
-        self.assertIn('data-user-create-comment="false"', html)
-        self.assertIn('data-user-create-subcomment="false"', html)
-
-    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
-    def test_discussion_render_successfully_with_orphan_parent(self, default_store):
```

Comment:
I'm not really seeing where this test and the next test were moved to.  Can you either help me find it, or help me understand why it might be a concern for modules but not xblocks?  Thank you.